apps/aecore/src/aec_fork_block_settings.erl

```diff
@@ -81,50 +61,17 @@ lima_contracts() -> preset_contracts(?LIMA_PROTOCOL_VSN,
                                      lima_contracts_file_missing).
 
 -spec accounts(aec_hard_forks:protocol_vsn()) -> list().
-accounts(ProtocolVsn) -> preset_accounts(accounts, ProtocolVsn, undefined).
+accounts(ProtocolVsn) -> preset_accounts(accounts, ProtocolVsn, accounts_file_missing).
 
 -spec extra_accounts(aec_hard_forks:protocol_vsn()) -> list().
-extra_accounts(ProtocolVsn) -> preset_accounts(extra_accounts, ProtocolVsn, undefined).
+extra_accounts(ProtocolVsn) -> preset_accounts(extra_accounts, ProtocolVsn, extra_accounts_file_missing).
 
 -spec contracts(aec_hard_forks:protocol_vsn()) -> list().
-contracts(ProtocolVsn) -> preset_contracts(ProtocolVsn, undefined).
-
-lookup_fork_config(Protocol, Key, Default) when is_integer(Protocol) ->
-    case aeu_env:config_value([<<"chain">>, <<"hard_forks">>], aecore, hard_forks, undefined) of
-        undefined ->
-            maybe_hardcoded_config(Protocol, Key, Default);
-        Map when is_map(Map) ->
-            case maps:get(integer_to_binary(Protocol), Map, undefined) of
-                undefined ->
-                    Default;
-                Height when is_integer(Height) ->
-                    if Key == <<"height">> ->
-                            Height;
-                       true ->
-                            maybe_hardcoded_config(Protocol, Key, Default)
-                    end;
-                Map1 when is_map(Map1) ->
-                    maps:get(Key, Map1, Default)
-            end
-    end.
-
-
-maybe_hardcoded_config(Protocol, Key, _Default) when is_integer(Protocol),
-                                                    Protocol =< ?CERES_PROTOCOL_VSN ->
-    case Key of
-        <<"use_hardcoded_directory">> -> true;
-        <<"directory">>               -> hardcoded_dir(Protocol)
-    end;
-maybe_hardcoded_config(_Protocol, Key, Default) ->
-    case Key of
-        <<"use_hardcoded_dir">> -> false;
-        _ ->
-            Default
-    end.
+contracts(ProtocolVsn) -> preset_contracts(ProtocolVsn, contracts_file_missing).
 
 -spec is_custom_fork() -> boolean().
 is_custom_fork() ->
-    is_map(aeu_env:user_map_or_env([<<"chain">>, <<"hard_forks">>], aecore, hard_forks, undefined)).
+    is_map(aeu_env:config_value([<<"chain">>, <<"hard_forks">>], aecore, hard_forks, undefined)).
```

Comment:
I'm not familiar with `aeu_env` implementation bug the function name change concerns me a bit, does it still considers ENV variables ?

Reply:
Just checked, seems to be the case still

Reply:
I didn't realise either. Ulf explains the difference in the pull request here: https://github.com/aeternity/aeternity/pull/4211

Reply:
Maybe I should change it back. There won't be any schema defaults.